frontend/src/components/MainNav.tsx

```diff
@@ -0,0 +1,14 @@
+import { Button } from './ui/button';
+
+const MainNav = () => {
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/10#issuecomment-2357741970>#5cbf44</a><span> - 09/18/2024, 07:52 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The MainNav component lacks flexibility as it doesn't accept any props, limiting its reusability
</div>
<div id="fix">
<b>Fix:</b> Modify the MainNav component to accept props for customization, such as button text, click handler, and additional CSS classes
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1,14 +1,23 @@
  import { Button } from './ui/button';
 +import React from 'react';
 +
 +interface MainNavProps {
 +  buttonText: string;
 +  onClick?: () => void;
 +  className?: string;
 +}
 
 -const MainNav = () => {
 +const MainNav: React.FC<MainNavProps> = ({ buttonText, onClick, className }) => {
    return (
      <Button
        variant="ghost"
 -      className="font-bold hover:text-orange-500 hover:bg-white"
 +      className={'font-bold hover:text-orange-500 hover:bg-white ${className || ''}'}
 +      onClick={onClick}
      >
 -      Log In
 +      {buttonText}
      </Button>
    );
  };
 
  export default MainNav;
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged